airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py

```diff
@@ -72,14 +72,6 @@ def __init__(
             s3_build_cache_secret_key=s3_build_cache_secret_key,
         )
 
-    @property
-    def docker_hub_username_secret(self) -> Secret:
-        return self.dagger_client.set_secret("docker_hub_username", self.docker_hub_username)
-
-    @property
-    def docker_hub_password_secret(self) -> Secret:
-        return self.dagger_client.set_secret("docker_hub_password", self.docker_hub_password)
-
```

Comment:
I like the fact that these 2 properties have been hoisted into the ConnectorContext but shouldn't the corresponding `@click.option`s either be defined at the `connectors` group level or for the `test` command? Right now they're only defined for the `publish` command IIRC.